requirements.txt

```diff
@@ -1 +1,3 @@
 async-timeout>=4.0.3
+cachetools
```

Comment:
We should restrict the versions of dependencies. We should also try to avoid adding `apscheduler` because it uses redis-py as an optional dependency.

Reply:
Fixed